serverless/event_sources/knative-event-sources.adoc

```diff
@@ -0,0 +1,31 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="knative-event-sources"]
+= Getting started with event sources
+include::modules/common-attributes.adoc[]
+:context: knative-event-sources
+toc::[]
+
+An _event source_ is a resource object, which is the link between an event producer and a _sink_ (or consumer) that receives those events. Currently, {ServerlessProductName} supports the following event source types:
+
+ApiServerSource:: Connects a sink to the Kubernetes API server.
+PingSource:: Periodically sends ping events with a constant payload. It can be used as a timer.
+
+SinkBinding is also supported, which allows you to connect core Kubernetes resources such as `Deployment`, `Job`, or `StatefulSet` with a sink.
+
+You can create and manage Knative event sources using the Developer perspective, the `kn` CLI, or by applying YAML files.
+
+.Prerequisites
+
+* You must have a current installation of xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#serverless-install-web-console_installing-openshift-serverless[{ServerlessProductName}], including Knative Serving and Eventing, in your {product-title} cluster. This can be installed by a cluster administrator.
+* You must create the `default` broker in the same namespace that the event source will be installed in. For more information, see xref:../../serverless/knative_eventing/serverless-using-brokers.adoc#serverless-using-brokers[Using brokers with Knative Eventing].
+* You must create or update a service account, role and role binding for event sources.
+* You must create a serverless application, or Knative service, before you can create an event source. Event sources need a service to use as an event _sink_. The sink is the service or application that events are sent to from the event source.
```

Comment:
you can create the source, but it does not fully reconcile.

Once the sink (e.g. Knative service) is there, the pod for the source will be created

Reply:
It doesn't let me create it at all without a service if you're using dev console? Is that expected behavior?

Reply:
I only know the yaml, and I can create it there, but the pod does not come up.

I guess the UX prevents this, as this might be undesired experience. @invincibleJai can u confirm?

Reply:
Assuming that this is the way it works/is supposed to work since we haven't heard anything. @invincibleJai let me know if this needs changes.